### PR TITLE
Translated to Brazilian Portuguese 

### PR DESCRIPTION
**Pull request by daguerpedro**
Created a 100% Portuguese readme and linked it to the main readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-Language English | [中文](/README-zhCN.md) | [한국어](/README-koKR.md) | [Español](/README-esES.md)
+Language English | [中文](/README-zhCN.md) | [한국어](/README-koKR.md) | [Español](/README-esES.md) | [Português](/README-ptBR.md)
```
Comment (Julgers) [inline README.md:5]: Could you perhaps add this to the other language READMEs as well? You added it to the English one only.

**Review by Julgers — CHANGES_REQUESTED** [review]
Thanks for taking the time to translate this, looks great. I have one little request before merging it in however.

File: README.md (hunk 1 above)
Comment (daguerpedro) [reply README.md:5]: Done

Comment (Julgers) [reply README.md:5]: Thanks!

**Review by Julgers — APPROVED** [review]
Approved.

**Pull request merged**